dcrsqlite/apisource.go

```diff
@@ -958,15 +958,19 @@ func (db *wiredDB) GetExplorerAddress(address string, count int) *explorer.Addre
 				NoOfUnconfirmed++
 			}
 			for _, y := range tx.Vout {
-				if address == y.ScriptPubKey.Addresses[0] {
-					t, _ := dcrutil.NewAmount(y.Value)
-					totalreceived += t
+				if len(y.ScriptPubKey.Addresses) != 0 {
+					if address == y.ScriptPubKey.Addresses[0] {
+						t, _ := dcrutil.NewAmount(y.Value)
+						totalreceived += t
+					}
 				}
 			}
 			for _, u := range tx.Vin {
-				if address == u.PrevOut.Addresses[0] {
-					t, _ := dcrutil.NewAmount(*u.AmountIn)
-					totalsent += t
+				if len(u.PrevOut.Addresses) != 0 {
```

Comment:
Note what I said about the panic.  Seems we also need `u.PrevOut!=nil`

Reply:
In addition to Len(u.PrevOut.Addresses) != 0?

Reply:
Yeah, turns out that PrevOut being nil was the issue

Reply:
Alright, would that be all before I make the next commit?

Reply:
I think so, but we'll keep iterating if needed.  No biggie.

`if u.PrevOut != nil && len(u.PrevOut.Addresses) > 0 {`